### PR TITLE
Update Japanese translation

### PR DESCRIPTION
**Pull request by luixxiul**
Based on #5915
Closes #5915

- Apply new strings
- Sort alphabetically

### CODE REVIEW

**Comment by saghul** [comment]
Thanks for taking this one! Let me know when it's ready so I can approve it.

File: lang/main-ja.json [hunk 1]
```diff
@@ -822,17 +822,17 @@
...
-        "videomute": "ビデオ停止中"
+        "videomute": "動画停止中"
     },
     "welcomepage": {
         "accessibilityLabel": {
             "join": "タップして参加",
             "roomname": "ルーム名を入力してください"
         },
-        "appDescription": "チームでビデオチャットしましょう。あなたの知り合いを招待してください。{{app}}は完全に暗号化された100％オープンソースのビデオミーティングソリューションで、いつでも無料でご利用いただけます。アカウントは必要ありません。",
+        "appDescription": "チームで動画チャットしましょう。あなたの知り合いを招待してください。{{app}}は完全に暗号化された100％オープンソースの動画ミーティングソリューションで、いつでも無料でご利用いただけます。アカウントは必要ありません。",
```
Comment (luixxiul) [inline lang/main-ja.json:832]: `動画チャット` sounds a bit unnatural. I'll revisit later.

File: lang/main-ja.json [hunk 2]
```diff
@@ -1,636 +1,781 @@
...
-        "inviteURLFirstPartGeneral": "",
-        "inviteURLFirstPartPersonal": "",
-        "inviteURLSecondPart": "",
+        "dialInNumber": "ダイヤルイン：",
+        "dialInSummaryError": "ダイヤルイン情報を取得できませんでした。後でもう一度お試しください。",
+        "dialInTollFree": "料金無料",
+        "genericError": "おっと、問題が発生しました。",
+        "inviteLiveStream": "このミーティングのライブストリームを表示するには、このリンクをクリックしてください：{{url}}",
+        "invitePhone": "電話で参加するには、これをタップしてください：{{number}},,{{conferenceID}}#\n",
+        "invitePhoneAlternatives": "別のダイヤルイン番号をお探しですか？\nミーティングのダイヤルイン番号をご参照ください：{{url}}\n\n\nルーム電話経由でダイヤルインしている場合は、オーディオに接続せずに参加してください：{{silentUrl}}",
```
Comment (luixxiul) [inline lang/main-ja.json:351]: original: https://github.com/jitsi/jitsi-meet/blob/6773aed67f08879333f16118c1953c485b443094/lang/main.json#L351

The translation is not perfect as context is unclear: needs more info for `If also dialing-in through a room phone, join without connecting to audio`

File: lang/main-ja.json [hunk 3]
```diff
@@ -1,636 +1,781 @@
...
+        "invalidStreamKey": "$t(dialog.liveStreaming)キーが誤っているかもしれません。",
+        "limitNotificationDescriptionNative": "$t(dialog.liveStreaming)は{{limit}}分に制限されています。無制限の$t(dialog.liveStreaming)には{{app}}をお試しください。",
+        "limitNotificationDescriptionWeb": "混雑のため$t(dialog.liveStreaming)は{{limit}}分に制限されています。無制限の$t(dialog.liveStreaming)には<a href={{url}} rel='noopener noreferrer' target='_blank'>{{app}}</a>をお試しください。",
+        "off": "$t(dialog.liveStreaming)が停止しました",
+        "offBy": "{{name}}が$t(dialog.liveStreaming)を停止しました",
+        "on": "$t(dialog.liveStreaming)",
+        "onBy": "{{name}}が$t(dialog.liveStreaming)を開始しました",
+        "pending": "$t(dialog.liveStreaming)を開始しています…",
+        "serviceName": "$t(dialog.liveStreaming)サービス",
+        "signedInAs": "あなたは現在、次のアカウントでログインしています。",
```
Comment (luixxiul) [inline lang/main-ja.json:422]: Context unclear: needs revist.

File: lang/main-ja.json (hunk 2 above)
Comment (luixxiul) [reply lang/main-ja.json:351]: Context: https://github.com/jitsi/jitsi-meet/issues/6403#issue-607355132

**Comment by luixxiul** [comment]
Closing. It does not make sense to me to keep update localization via PR, while original strings are added to the source file, main.json, without taking localization files into consideration. Just please restart https://translate.jitsi.org. https://github.com/jitsi/jitsi-meet/issues/5056

**Pull request closed without merging**